docs/configuration.md

```diff
@@ -456,6 +456,33 @@ Apart from these, the following properties are also available, and may be useful
     from JVM to Python worker for every task.
   </td>
 </tr>
+<tr>
```

Comment:
All the SQL configurations should follow what we did in the section of `Spark SQL` https://spark.apache.org/docs/latest/configuration.html#spark-sql.

Reply:
These confs are not part of `spark.sql("SET -v").show(numRows = 200, truncate = false)`.

Reply:
Got it, I'll add there configurations into SQLConf.scala in the follow up pr.